lib/bundles.cf

```diff
@@ -382,6 +382,47 @@ bundle agent cmerge(name, varlist)
       "$(name)_str" string => format("%S", $(name)),    policy => "free";
 }
 
+bundle agent collect(name, tag)
```

Comment:
Since we are collecting values, do you think it would make more sense to call the bundle collect_values ?